packages/lesswrong/lib/executionEnvironment.ts

```diff
@@ -20,7 +20,6 @@ export const isAnyTest = bundleIsTest
 export const isPackageTest = bundleIsTest
 
 export interface CommandLineArguments {
-  mongoUrl: string
```

Comment:
Did you intend to completely remove mongo as an option in this PR (reasonable thing to do). If so then there are quite a few places where it's still reference, such as in ./build.js where it is listed as a valid command line option